hot-reload-analysis/src/main/kotlin/org/jetbrains/compose/reload/analysis/InstructionToken.kt

```diff
@@ -455,3 +453,14 @@ internal fun tokenizeInstructions(
 
     return tokens.toLeft()
 }
+
+
+private fun TokenizerContext.Consumer.findMethodInsn(): MethodInsnNode? {
+    return find { node ->
+        when (node) {
+            is MethodInsnNode -> node
+            is LabelNode, is LineNumberNode -> null
```

Comment:
What is the motivation for explicitly checking `LabelNode` and `LineNumberNode` here? If you want to keep it, maybe we can add a comment to explain it?

Reply:
Those two node types can effectively be 'ignored', so we scan until we find either a `MethodInsnNode` , consuming the 'noise'